source/content/guides/decoupled-sites/site.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Pantheon Decoupled
+subtitle: 
+description: 
+categories: [platform]
+tags: [webops, workflow, decoupled]
+contributors: [joa-pan]
+reviewed: "2022-07-31"
+layout: guide
+showtoc: true
+permalink: docs/guides/decoupled-sites/site
+anchorid: site
+---
+
+# Before You Begin
+
+You should have access to the Early Access Customer checklist provided during onboarding. To begin using Pantheon Decoupled, ensure you meet the following prerequisites:
+
+* Your repository is in GitHub.
+
+* You have a domain to use for the decoupled site.
+
+* You will use Next.js or Gatsby as the Frontend framework.
+
+* (Optional) You are using Drupal 9 or WordPress for the CMS.
+
+* (Optional) If using a preconfigured starter kit template, you will use one of the following combinations:
+
+    * Drupal 9 and Next.js
+
+    * WordPress and Gatsby
+
+    Note: Other configurations are not currently supported. 
+
+
+
+## Site Setup
+
+Pantheon Decoupled is available through the New Dashboard. To access it log in to your Pantheon Dashboard and Navigate to the Sites page. 
+
+. Select **Sites** in the the New Dashboard. 
+
+. Click the **Decoupled** tab to view the Early Access offering. 
+
+. Click **+Create New Site** in the New Dashboard  under the **Decoupled** tab to begin your decoupled journey. 
+
+
+You can create an application from scratch using **Importing a repository** or you can use our starter kit template for:
+
+* Server side render applications: Next.js and Drupal
```

Comment:
Hi @CobyPear - is it correct to refer to the Next.js and Drupal as the  serverside render (SSR) applications and Gatsby and WordPress as the SSG

Reply:
Currently on the infra, Gatsby is SSG only
Next.js does do both SSR and SSG, however it's not SSG in the same way as it's not being served the same way. But it does still conform to the SSG definition. Unfortunately at the moment that's the answer albeit confusing. Good news is that in the long term the frameworks should have all of their rendering methods supported, not yet tho.